llama_index/utils.py

```diff
@@ -81,7 +81,11 @@ def stopwords(self) -> List[str]:
             try:
                 nltk.data.find("corpora/stopwords")
             except LookupError:
-                nltk.download("stopwords")
+                nltkDataFolder = os.environ.get("NLTK_DATA")
+                if nltkDataFolder is None:
+                    nltk.download("stopwords")
+                else:
+                    nltk.download("stopwords", download_dir=nltkDataFolder)
```

Comment:
you can simplfy this a bit actually. Also, lets use snake case

```python
nltk_data_dir = os.environ.get("NLTK_DATA", None)
nltk.download("stopwords", download_dir=nltk_data_dir )
```

Reply:
Thank you, @logan-markewich !